docs/en/connector-v2/sink/GoogleSheets.md

```diff
@@ -0,0 +1,55 @@
+# GoogleSheets
+
+> GoogleSheets sink connector
+## Description
+
+Used to write data to GoogleSheets.
+
+## Key features
+
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+- [ ] [schema projection](../../concept/connector-v2-features.md)
```

Comment:
`schema projection` is not sink key features. Please reference https://seatunnel.apache.org/docs/concept/connector-v2-features.